QuizBowlDiscordScoreTracker/Commands/AdminCommands.cs

```diff
@@ -17,10 +17,11 @@ public AdminCommands(IDatabaseActionFactory dbActionFactory)
         private IDatabaseActionFactory DatabaseActionFactory { get; }
 
         [Command("checkPermissions")]
-        [Summary("Checks if the bot has all the required permissions.")]
-        public Task CheckPermissionsAsync()
+        [Summary("Checks if the bot has all the required permissions. " + 
+                 "Omitting a channel mention will check the current channel's permissions.")]
+        public Task CheckPermissionsAsync([Summary("Text channel mention (#textChannelName)")] ITextChannel messageChannel)
```

Comment:
The argument needs to be [Optional], otherwise this will always require a channel

Reply:
Now reads `... ITextChannel messageChannel = null`
If there is a different way to specify an Optional here, please specify

Reply:
Use the [Optional] attribute on the argument, next to the [Summary] one. Optional lives in the System.Runtime.InteropServices namespace, so you'll have to include that namespace in the usings.

Reply:
That said, it appears defaulting messageChannel to null also works, so this approach is fine too.

Reply:
I'll change this to use the [Option] attribute from the specified namespace. I wasn't sure if I could import from there (if InteropServices had been included in the project at all).